androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/Utils.java

```diff
@@ -99,4 +102,41 @@ public static Bitmap convertDrawableToBitmap(Context context, int drawableId) {
         drawable.draw(canvas);
         return bitmap;
     }
-}
\ No newline at end of file
+
+    /**
+     * Verifies if the application has the required meta-tag, `web_manifest_url`, and the
+     * `web_app_manifest.json` file, required to enable the Trusted Web Activity on Chrome OS.
+     * Important: The content the meta-tag and the JSON fire are not validated.
+     */
+    public static boolean hasChromeOsSupport(Context context) {
+        // We first check if /app/res/web_app_manifest.json is available
+        int id = context.getResources().getIdentifier(
+                "web_app_manifest.json", "raw", context.getPackageName());
+        // If an attribute with that name doesn't exist the result is 0 (which is an invalid ID for
+        // attributes)
+        if (id == 0) {
+            Log.e("ChromeOS-Support", "Could not find '/res/raw/web_app_manifest.json'");
```

Comment:
I think this qualifies as a warning rather than an error - aren't there valid cases where TWAs don't want to do this on ChromeOS?

Reply:
Agreed - changed to warnings.